Library/Formula/magicrescue.rb

```diff
@@ -0,0 +1,14 @@
+class Magicrescue < Formula
+  desc "Magic Rescue extracts files of known types from a block device."
```

Comment:
You might want to write a `desc` that doesn’t start with “Magic Rescue”. `brew desc magicrescue` will show the following:
```
magicrescue: Magic Rescue extracts files of known types from a block device.
```